contributing/scc.txt

```diff
@@ -71,24 +71,50 @@ are queried at the prompt::
 scc merge
 ---------
 
-.. program:: scc merge
-
 Merge all the PRs based on specified branch matching the input filters
 including all submodules.
 
+Description
+^^^^^^^^^^^
+
+Filters of different types can be specified and combined to include and
+exclude a set of PRs in the merge command. Each filter needs to be formatted
+as `key:value`. If no key but only a value is specified, it is assumed the
+filter is a label filter (see below). These filters can be passed to an :option:`scc merge --include` or :option:`scc merge --exclude` option.
+
+The available filter types are described below:
+
+- Label filters can be specified using the `label` key i.e. `label:<LABEL>`.
+  This filter type will select a Pull Request if one the following condition
+  is met in order of precedence:
```

Comment:
Right, I might remove this. The ordering only really matters as an implementation detail.